### PR TITLE
Gateway airdrop

### PR DESCRIPTION
**Pull request by CAPtheorem**
Add UI for airdrop. Working. Known bugs - calling 
```
const response = await omgxWatcherAxiosInstance(
        this.masterSystemConfig
      ).post('send.l2.airdrop', {
          address: this.account,
          key: process.env.REACT_APP_AIRDROP
      })
```
does not yet change the claimed flag and set the timestamp

minor - also need to update the airdrop menu icon

### CODE REVIEW

**Review by cby3149 — APPROVED** [review]

File: packages/boba/gateway/src/containers/airdrop/Airdrop.js [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+            </Typography>
+          }
+
+          {recordFoundL2 && snapValueL2 > 0 && claimedL2 === false &&
+            <>
+              <Typography 
+                variant="body2" 
+                component="p" 
+              >
+                There was a balance of {snapValueL2*0.95} OMG on Boba during the snapshot.
```
Comment (souradeep-das) [inline packages/boba/gateway/src/containers/airdrop/Airdrop.js:326]: should this be more accurate?
> {snapValueL2*0.95}

snapValueL2 * (100/105)

Comment (CAPtheorem) [reply packages/boba/gateway/src/containers/airdrop/Airdrop.js:326]: Yes good eye! Fixing

File: packages/boba/gateway/src/containers/airdrop/Airdrop.js [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+          {/* STATE 3 - OMG ON L1 DURING SNAPSHOT AND NOT CLAIMED AND NOT INITIATED BUT NOT ENOUGH OMG ON L2 */}
+          {recordFoundL1 && (snapValueL1 > 0) && (l2BalanceOMG <= snapValueL1 * 0.97) && (claimedL1 === false) && (unlockL1time === 0) &&
+            <Typography 
+              variant="body2" 
+              component="p" 
+              sx={{color: 'yellow'}}
+            >
+              Yes, there was a balance of {snapValueL1} OMG on Ethereum during the snapshot.
+              <br/>However, your current OMG balance on Boba is only {l2BalanceOMG}. 
+              <br/>Please bridge {(snapValueL1 - l2BalanceOMG)*0.99} or more OMG to Boba to claim your airdrop.
```
[the remark this answers is not in the document]
Comment (CAPtheorem) [reply packages/boba/gateway/src/containers/airdrop/Airdrop.js:240]: Yes that was the idea - change?

File: packages/boba/gateway/src/containers/airdrop/Airdrop.js (hunk 1 above)
Comment (CAPtheorem) [reply packages/boba/gateway/src/containers/airdrop/Airdrop.js:326]: Let's go with that

File: packages/boba/gateway/src/containers/airdrop/Airdrop.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (CAPtheorem) [reply packages/boba/gateway/src/containers/airdrop/Airdrop.js:240]: fixed

**Pull request merged**